apps/E2E/src/Callout/pages/CalloutPageObject.win.ts

```diff
@@ -14,32 +14,20 @@ class CalloutPageObject extends BasePage {
     return await (await this._primaryComponent).isDisplayed();
   }
 
-  /* OVERRIDE: This must scroll to the button that opens the callout, not the callout (since it's not visible on load.) */
-  async scrollToTestElement(): Promise<void> {
-    // on win32, adding a blank value to an element automatically scrolls to it
-    await (await this._buttonToOpenCallout).addValue('');
-  }
-
-  async openCallout(): Promise<void> {
+  // This both opens and waits for it to go in view
+  async openCalloutAndWaitForLoad(): Promise<void> {
     await (await this._buttonToOpenCallout).click();
+    await this.waitForCondition(
+      async () => await this.didCalloutLoad(),
+      'The testing callout failed to display after attempting to open it.',
```

Comment:
Nit: Let's get more specific:

"Clicked on the button to open the Callout, but the Callout did not open correctly"